app/views/coronavirus_landing_page/components/shared/_meta_tags.html.erb

```diff
@@ -5,9 +5,10 @@
 <% content_for :is_full_width_header, true %>
 <% content_for :title, @content_item["title"] %>
 <% content_for :meta_tags do %>
+  <% page_url = request.base_url + request.path %>
```

Comment:
I'd suggest using `File.join` to handle the slash placement.

Reply:
👍